usr/local/www/diag_dns.php

```diff
@@ -91,7 +95,7 @@
 
 if ($_POST) {
 	unset($input_errors);
-
+	
```

Comment:
Please configure your IDE to strip trailing whitespace, otherwise we'd have to keep running clean.sh on all commits ;)